packages/jest-cli/src/__tests__/TestRunner-test.js

```diff
@@ -38,6 +38,115 @@ test('.addReporter() .removeReporter()', () => {
   expect(runner._dispatcher._reporters).not.toContain(reporter);
 });
 
+describe('noDefaultReporters option', () => {
+  let runner;
+  
+  test('does not add reporters if true', () => {
+    runner = new TestRunner({}, {noDefaultReporters: true}, {});
+    expect(runner._dispatcher._reporters).toHaveLength(0);
+  });
+
+  test('adds reporters if undefined', () => {
+    runner = new TestRunner({}, {}, {});
+    expect(runner._dispatcher._reporters).toHaveLength(2);
+  });
+
+  test('adds reporters if false', () => {
+    runner = new TestRunner({}, {noDefaultReporters: false}, {});
+    expect(runner._dispatcher._reporters).toHaveLength(2);
+  });
+});
+
+// Checking for the method _addCustomReporters
+// Custom reporters used here are the reporters within the package
+// No extra reporters are included to be used
+describe('_addCustomReporters', () => {
```

Comment:
this method is prefixed with `_` which signifies that it's private. Generally we try to avoid testing private methods (although in rare cases it's necessary).
Instead of writing many unit tests for these modules i'd encourage you to write a few integration tests (you can check `jest/integration_tests/` directory to see the examples.

Jest integration tests run Jest agains an existing example directory and verify that `stdout/stderr` output matches what's expected. They are a little slow and hard to maintain, but they provide much more value and catch more bugs. At the same time you don't have to change anything in your tests when you refactor code, sinc you're not dealing with implementation details.